proposals/0161-StringRevision1.md

```diff
@@ -0,0 +1,342 @@
+# String Revision: Collection Conformance, C Interop, Transcoding
+
+* Proposal: [SE-0161](0161-StringRevision1.md)
```

Comment:
`0161` is already taken (by the *Smart KeyPaths* proposal). All existing filenames are lowercased.

```diff
-* Proposal: [SE-0161](0161-StringRevision1.md)
+* Proposal: [SE-NNNN](NNNN-string-revision-1.md)
```